src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -32,14 +21,16 @@ import {
   useValue,
 } from '../../hooks';
 import {
+  AutoCompleteValue,
   ChangeEvent,
   UISchemaElementComponent,
   UISchemaElementComponentWithValidationProps,
 } from '../../types';
 import { getTranslation } from '../../util';
-import FieldLevelMessageContainer from '../FieldLevelMessageContainer';
 import { withFormValidationState } from '../hocs';
 
+type PasswordAutoCompleteValue = Extract<AutoCompleteValue, 'current-password' | 'new-password'> | undefined;
+
```

Comment:
Odyssey scoped down their typing for `PasswordField`'s `autoCompleteValue` prop so this type makes it nicer for us to type cast